operations/charge.go

```diff
@@ -51,6 +51,12 @@ func (req *ListCharges) Describe() *internal.Description {
 //
 //	fmt.Println("created charge:", charge.ID)
 //
+
+type PlatformFee struct {
+	Percentage float64 `json:"percentage,omitempty"`
+	Fixed      string  `json:"fixed,omitempty"`
```

Comment:
Is it ok that this type is `string` because the types of `amount` are used `int64` in other points.

Reply:
Good point, I'll fix this.